tests/tests_ssh.yml

```diff
@@ -11,12 +10,13 @@
     - name: gather facts from {{ kdump_ssh_server_outside }}
       setup:
       delegate_to: "{{ kdump_ssh_server_outside }}"
-      delegate_facts: True
+      delegate_facts: true
 
     - include_role:
         name: kdump
       vars:
-        kdump_ssh_user: "{{ hostvars[kdump_ssh_server_outside]['ansible_user_id'] }}"
+        kdump_ssh_user: |
+          {{ hostvars[kdump_ssh_server_outside]['ansible_user_id'] }}
```

Comment:
Unfortunately, this is not an equivalent change. It adds a newline after the template.
Given this yaml file
```yaml
singleline: "{{ hostvars[kdump_ssh_server_outside]['ansible_user_id'] }}"
multiline: |
  {{ hostvars[kdump_ssh_server_outside]['ansible_user_id'] }}
```
the following code:
```python
import yaml
y = yaml.safe_load(open('yaml_multiline.yml', 'r'))
print(y['singleline'])
print('---')
print(y['multiline'])
print('---')
```
prints:
```
{{ hostvars[kdump_ssh_server_outside]['ansible_user_id'] }}
---
{{ hostvars[kdump_ssh_server_outside]['ansible_user_id'] }}

---
```
(note the empty line after the second template).
Do we have lots of places where such changes were made?

Reply:
>  Do we have lots of places where such changes were made?

possibly several

What about this:
```yaml
multiline: >
  {{ hostvars[kdump_ssh_server_outside]['ansible_user_id'] }}
```
?

Reply:
This works:
```yaml
multiline: >-
  {{ hostvars[kdump_ssh_server_outside]['ansible_user_id'] }}
# OR
multiline: |-
  {{ hostvars[kdump_ssh_server_outside]['ansible_user_id'] }}
```

Reply:
I have found the `>-` form independently by digging trough the YAML spec. In particular https://yaml.org/spec/1.1/#chomping/ . :+1: 
I don't think we want `|-` for those cases where we simply want to split a line that is too long, although it would work here as well.

Reply:
It is also the method used in the template: https://github.com/linux-system-roles/template/blob/52377c9c8a02493d4e729cc15d4749e7f69dfe3e/tasks/main.yml#L13 which is good.

Reply:
PR #63